test/ngSanitize/sanitizeSpec.js

```diff
@@ -280,10 +280,51 @@ describe('HTML', function() {
     expect(doc).toEqual('<p><img src="x"></p>');
   }));
 
+  describe('Custom white-list support', function() {
+    beforeEach(module(function($sanitizeProvider) {
+      $sanitizeProvider.addValidElements(['form']);
```

Comment:
In order to make sure this test does indeed test what it is supposed to (now and in the future), it might be better to add arbitrary elements (that do not exist in the HTML5 spec) to ensure that they are not already considered valid.